entries/n/nimbusweb.me.json

```diff
@@ -1,12 +1,14 @@
 {
-  "Nimbus Platform": {
+  "FuseBase": {
     "domain": "nimbusweb.me",
```

Comment:
They changed name but not domain name? 🤨

Reply:
It's been 1.5 years since the rebranding was announced (https://nimbusweb.me/blog/nimbus-becomes-fusebase/), but still the account itself and a Google search gives a link to the old domain, and the new domain has no stats even on Similarweb: https://www.similarweb.com/website/thefusebase.com

I added https://thefusebase.com/ as `url`, I think that will be enough.

Reply:
Ah okay.
I think it would be better to add thefusebase.com as an additional domain. 🙂

Reply:
👍 Done: 7c17077